packages/frontend/src/store/timeEntries.ts

```diff
@@ -1,8 +1,9 @@
 import { State } from "./index";
 import { ActionContext } from "vuex";
-import { debounce } from "lodash";
+import { debounce, entries } from "lodash";
```

Comment:
Usikker på hvordan entries har sneket seg med her, men den brukes ikke :)

```suggestion
import { debounce } from "lodash";
```